src/messages/MMgrBeacon.h

```diff
@@ -147,7 +147,17 @@ class MMgrBeacon final : public PaxosServiceMessage {
 
     encode(modules, payload);
     encode(mgr_features, payload);
-    encode(clients, payload, features);
+
+    std::vector<entity_addrvec_t> clients_addrs;
+    for (const auto &i : clients) {
```

Comment:
(Throughout) Keep the ref on the type for consistency:
```suggestion
    for (const auto& p : clients) {
```